OCPBUGS#57348: release_notes/ocp-4-19-release-notes: Add a boot-image-clobber known issue (#94987)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

* In {product-title} {product-version}, clusters using IPsec for network encryption might experience intermittent loss of pod-to-pod connectivity. This prevents some pods on certain nodes from reaching services on other nodes, resulting in connection timeouts. Internal testing could not reproduce this issue on clusters with 120 nodes or less. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-55453[OCPBUGS-55453])

> * {product-title} clusters that are installed on {aws-short} with custom AMIs or on {gcp-short} with custom disk images will have those customizations overridden by boot image management. To recover, you must xref:../machine_configuration/mco-update-boot-images.adoc#mco-update-boot-images-disable_machine-configs-configure[disable boot image management], restore your MachineSet boot images, and delete any Machines created with an undesired boot image.(link:https://issues.redhat.com/browse/OCPBUGS-57348[OCPBUGS-57348])

Reviewer: ```suggestion
* If you install a cluster on {aws-short} that has Amazon Machine Images (AMI) enabled or on  {gcp-short} that has custom disk images enabled, the boot image management overrides these customization images with boot images.  As a workaround, you can disable the boot image management feature, restore the boot images for the machine sets to their original location, and delete any machines that were incorrectly generated by the overriding boot images. (link:https://issues.redhat.com/browse/OCPBUGS-57348[OCPBUGS-57348])
```

Author: I have some concerns with that suggestion, but in the interest of getting _something_ declared as a known-issue, I've adopted your wording (and pivoted from the 4.20 OCPBUGS-57348 to the 4.19.z OCPBUGS-57796, now that that backport tracker exists) with 72d6586371 -> cf256d9e38.  See the cf256d9e38 commit message for more on my concerns.  I'll leave it up to docs folks if any of them are worth follow-up pull requests.

/label merge-review-needed

> * If you install a cluster on {aws-short} that has Amazon Machine Images (AMI) enabled or on  {gcp-short} that has custom disk images enabled, the boot image management overrides these customization images with boot images.  As a workaround, you can disable the boot image management feature, restore the boot images for the machine sets to their original location, and delete any machines that were incorrectly generated by the overriding boot images. To disable boot image management, see ref:../machine_configuration/mco-update-boot-images.adoc#mco-update-boot-images-disable_machine-configs-configure[disable boot image management]. (link:https://issues.redhat.com/browse/OCPBUGS-57796[OCPBUGS-57796])

Reviewer 2: ```suggestion
If you install a cluster on {aws-short} with Amazon Machine Images (AMI) enabled, or on {gcp-short} with custom disk images enabled, the boot image management feature overrides these custom images with default boot images.
+
As a workaround, you can disable the boot image management feature, restore the original boot images for the machine sets, and delete any machines that were incorrectly created by the overriding boot images.
+
To disable boot image management, see ref:../machine_configuration/mco-update-boot-images.adoc#mco-update-boot-images-disable_machine-configs-configure[Disable boot image management].
+
(link:https://issues.redhat.com/browse/OCPBUGS-57796[OCPBUGS-57796])
```

I'm thinking something like this? Be sure to remove doubled spaces and clarify the first clause, regardless.

Author: > I'm thinking something like this?

Fine with me; pushed with cf256d9e38 -> a35fd43734.  I also have `Allow edits and access to secrets by maintainers` checked, so folks can push the rewords themselves, or close my pull and open your own?  I really don't care what the words are here, it just seems like an issue that is worth a known-issue declaration.  If there's a way I can delegate known-issue declaration to someone who has more time to shepherd the change through to landing, please let me know.
